sopel/modules/help.py

```diff
@@ -75,13 +112,17 @@ def help(bot, trigger):
             if not url:
                 return
             bot.memory['command-list'] = (len(bot.command_groups), url)
+            update_cache(bot)
         bot.say("I've posted a list of my commands at {0} - You can see "
                 "more info about any of these commands by doing {1}help "
                 "<command> (e.g. {1}help time)".format(url, help_prefix))
 
 
 def create_list(bot, msg):
-    msg = 'Command listing for {}@{}\n\n'.format(bot.nick, bot.config.core.host) + msg
+    msg = 'Command listing for {}'.format(bot.nick) \
+        + ('@{}'.format(bot.config.core.host) if bot.config.help.show_server_host else '') \
```

Comment:
This can be done in one `.format()` call, and within parentheses, avoiding the need for `\` to continue lines.

Reply:
:+1: